packages/math/atoms.lua

```diff
@@ -0,0 +1,38 @@
+local atomType = {
+   ordinary = 0,
+   bigOperator = 1,
```

Comment:
Very misleading from the start: we initially conflated big ops and other "mathop". It's true that both _may_ have largeop or movablelimits... But there's more to it here: `\sin` in TeX would be a mathop too, and proper spacing then occurs with but a small fix (due to how paired open/close logic, TeX does ofc things differently)
![image](https://github.com/user-attachments/assets/8874c3a1-d660-4468-a041-04e8b906bcd3)

Reply:
Done, and a test case added.